src/samples/Uno.Material.Samples/Uno.Material.Samples.Shared/Content/NestedSamples/CommandBarSample_NestedPage1.xaml.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices.WindowsRuntime;
+using Windows.Foundation;
+using Windows.Foundation.Collections;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Controls.Primitives;
+using Windows.UI.Xaml.Data;
+using Windows.UI.Xaml.Input;
+using Windows.UI.Xaml.Media;
+using Windows.UI.Xaml.Navigation;
+
+namespace Uno.Material.Samples.Content.NestedSamples
+{
+	public sealed partial class CommandBarSample_NestedPage1 : Page
+	{
+		public CommandBarSample_NestedPage1()
+		{
+			this.InitializeComponent();
+
+			this.SeeCodeBehindButton.Content = GetCodeBehind();
+		}
+
+		private string GetCodeBehind()
+		{
+			return @"
+private void NavigateToNextPage(object sender, RoutedEventArgs e) =>
+	Frame.Navigate(typeof(CommandBarSample_NestedPage2));
+
+private void NavigateBack(object sender, RoutedEventArgs e)
+{
+	// Normally we would've just called `Frame.GoBack();` if we only have a single frame.
```

Comment:
Since it's already mocked code, we can just show Frame.GoBack() ? and avoid the explanation :) What do you think?